test/ideas.list.js

```diff
@@ -830,4 +817,195 @@ describe('read lists of ideas', () => {
       });
     });
   });
+
+  describe('GET /ideas?filter[trending]', () => {
+    let user0,
+        user1,
+        user2,
+        user3,
+        user4,
+        user5,
+        user6,
+        user7,
+        user8,
+        idea1,
+        idea2,
+        idea3,
+        idea4,
+        idea5,
+        idea6;
+    const now = new Date();
+    let sandbox, clock;
+    const threeMonths = 7776000000;
+    const threeWeeks = 1814400000;
+    const oneWeek = 604800000;
+    const twoDays = 172800000;
+    // create and save testing data
+    beforeEach(async () => {
+      sandbox = sinon.sandbox.create();
+      const primarys = 'ideas';
+      const data = {
+        users: 10,
+        verifiedUsers: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+        ideas: Array(11).fill([]),
+        // odeas with votes: 3:3, 1:3, 5:1, 2:1, 0:0, 6: -1, 4:-2
+        votes: [
+          [0, [primarys, 1], 1],
+          [0, [primarys, 2], 1],
+          [0, [primarys, 3], 1],
+          [1, [primarys, 3], 1],
+          [2, [primarys, 3], 1],
+          [0, [primarys, 4], 1],
+          [1, [primarys, 4], 1],
+          [2, [primarys, 4], 1],
+          [3, [primarys, 4], 1],
+          [4, [primarys, 4], 1],
+          [0, [primarys, 5], 1],
+          [1, [primarys, 5], 1],
+          [2, [primarys, 5], 1],
+          [3, [primarys, 5], 1],
+          [4, [primarys, 5], 1],
+          [5, [primarys, 5], 1],
+          [6, [primarys, 5], 1],
+          [0, [primarys, 6], 1],
+          [1, [primarys, 6], 1]
+        ]
+      };
+      // post initial data and oldest votes with date three monts ago without two days
+      clock = sinon.useFakeTimers(now.getTime() - threeMonths + twoDays);
```

Comment:
Nice work with the fake timers to populate the database! Perhaps we could later tweak the [./test/handle-database.js](https://github.com/ditup/ditapi/blob/master/test/handle-database.js#L330) so the `created` timestamp could be hardcoded. However it is clear as it is.